lib/kcerts/ssh.go

```diff
@@ -96,9 +97,27 @@ func (a SSHAgent) Valid() bool {
 	return err == nil
 }
 
+func (a SSHAgent) AddCertificates(privateKey *rsa.PrivateKey, publicKey ssh.PublicKey, ttl uint32) error {
+	conn, err := net.Dial("unix", a.Socket)
+	if err != nil {
+		return err
+	}
+	defer conn.Close()
+	cert, ok := publicKey.(*ssh.Certificate)
```

Comment:
You can pass in a ssh.Certificate in the publicKey section of this function, considering ssh.PubKey is an interface. The intention behind accepting the interface is because it's standard to accept interfaces and means the caller doesn't have to typecast considering there's no way to marshal a certificate (always have to typecast a pub key).

Reply:
ack, no strong feelings here.